app/src/main/java/xyz/malkki/neostumbler/MainActivity.kt

```diff
@@ -53,14 +55,20 @@ class MainActivity : AppCompatActivity() {
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        val dynamicColorFlow = useDynamicColor().stateIn(lifecycleScope, started = SharingStarted.Eagerly, initialValue = null)
+        val dynamicColorFlow = useDynamicColor().stateIn(
+            lifecycleScope,
+            started = SharingStarted.Eagerly,
+            initialValue = null,
+        )
 
         installSplashScreen().apply {
             setKeepOnScreenCondition {
                 dynamicColorFlow.value == null
             }
         }
 
+        enableEdgeToEdge()
```

Comment:
What does this do? On the emulator the only difference I can see is that the system navigation bar is white instead of using the app theme

Reply:
Yes, and it looks much nicer!

See:
- https://developer.android.com/develop/ui/views/layout/edge-to-edge
- https://developer.android.com/about/versions/15/behavior-changes-15#edge-to-edge

In other apps you may have componets you need to adapt and manually set some insets but since the only components that NeoStumbler has that come close to the edge are TopAppBar and NavigationBar, and they naively handle edge-to-edge, you don't need to do anything else.

E.g. see:
- https://gitlab.com/fdroid/fdroidclient/-/merge_requests/1482
- https://gitlab.com/Nulide/findmydevice/-/merge_requests/238

Reply:
See the difference here:
1. With `enableEdgeToEdge()`:
![image](https://github.com/user-attachments/assets/7f182c89-5897-426a-bb25-d36cd5a2308f)
2. Without `enableEdgeToEdge()`:
![image](https://github.com/user-attachments/assets/31dad75c-a95f-4ab0-896d-f863ce92a1d4)

I'm not sure what it does, but the one without `enableEdgeToEdge()` looks like the correct one to me

Reply:
Oof, I didn’t try 3-button navigation. In that case, turns out that Compose doesn’t support it so great out of the box as it claims. :/

This is what it looks like on my Android 14 device:

| With edgeToEdge  | Without EdgeToEdge |
|-|-|
|<img src=https://github.com/user-attachments/assets/e4b41498-eeae-4bc8-a0b7-0c0be4f5e87e width=300>|<img src=https://github.com/user-attachments/assets/a43673f1-8703-4e41-a358-edcb77eccdab width=300>